website/docs/static.md

```diff
@@ -81,6 +81,19 @@ which errors are reported.
 > a signature in a `# typed: false` file might introduce new type errors if it's
 > called from a `# typed: true` file.
 
+### Upgrading a file from `ignore` to any other sigil
+
+After changing the sigil of an `ignore`d file, you must run `srb rbi 
+hidden-definitions`.
+
+The last time hidden definitions were generated, the file was `ignore`d, sorbet 
+didn't load it, so anything in it was treated as "hidden" and defined in 
+`sorbet/rbi/hidden-definitions/hidden.rbi`. Now that sorbet
+is allowed to load it, those definitions must be removed from `hidden.rbi`.
+
+If you don't regenerate `hidden.rbi`, you are likely to encounter [error 
```

Comment:
```suggestion
If you don't regenerate `hidden.rbi`, you could encounter [error 
```

Reply:
The only way you *wouldn't* get an error 4010 is if the previously ignored file was empty, right? So, the word "likely" seems appropriate here.  If I'm right about the probability, I think the docs should express that. Please let me know if I'm wrong.